Create ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md (#2238)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-2-13-adjust-widget-title-and-actions-for-overlay-and-hidden-appearance-6643.md

---
date: ""

> title: Widget title and actions adjusted for overlay and hidden appearance

Reviewer: ```suggestion
title: Widget title and actions aligned with overlay and hidden appearance settings
```

> title: adjust widget title and actions for overlay and hidden appearance (#6643) [GRAFT][release/cd] (#6656)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Widget title and actions adjusted for overlay and hidden appearance
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-59816
version: 1020.2.13
---

> adjust widget title and actions for overlay and hidden appearance (#6643) [GRAFT][release/cd] (#6656)

Author: The Claude AI suggested to use the following body:
```suggestion
To improve the user experience when configuring widgets, the widget title and action buttons have been adjusted for the overlay and hidden widget appearance settings. Previously, the title and action buttons were always shown which could be confusing in some configurations. With this change, the title and action buttons are now hidden when either the overlay or hidden appearance is selected in the widget settings. This impacts the widget configuration for all users, but the actual widget appearance and behavior for end users remains unchanged.
```

> The display of widget titles and action buttons has been optimized to align with the overlay and hidden widget appearance settings. Previously, these elements were always visible, which could be confusing when edit widgets mode was disabled. With this update, the action button is now only displayed when either the overlay or hidden appearance is selected in the widget settings and edit widgets mode is enabled. 

Reviewer: ```suggestion
The display of widget titles and action buttons has been optimized to align with the overlay and hidden widget appearance settings. Previously, these elements were always visible, which could be confusing when the edit widgets mode was disabled. With this update, the action button is now only displayed when either the overlay or hidden appearance is selected in the widget settings and the edit widgets mode is enabled. 
```
